spec/models/manageiq/providers/workflows/automation_manager/workflow_instance_spec.rb

```diff
@@ -55,10 +51,44 @@
   end
 
   describe "#run" do
-    it "test" do
+    it "sets the status to success" do
       workflow_instance.run
 
       expect(workflow_instance.reload.status).to eq("success")
     end
+
+    context "with a Credentials property in the workflow_content" do
+      let(:credentials) { {"username" => "$.my-credential.userid", "password" => "$.my-credential.password"} }
```

Comment:
My only concern here is that it's not really discoverable at the moment, but I can't think of anything better.

Reply:
There is a `GET /api/authentications` endpoint with `embedded_automation_manager_credentials_*` role permissions would that work?